packages/fx-core/tests/plugins/resource/aad/unit/generateArmTemplates.test.ts

```diff
@@ -22,159 +22,162 @@ describe("AadGenerateArmTemplates", () => {
 
   it("generate arm templates: tab", async function () {
     // Act
-    const activeResourcePlugins = [
-      ResourcePlugins.Aad,
-      ResourcePlugins.FrontendHosting,
-      ResourcePlugins.SimpleAuth,
-    ];
-    const pluginContext: PluginContext = await TestHelper.pluginContext(new Map(), true, false);
-    pluginContext.projectSettings!.solutionSettings = {
-      name: "test_solution",
-      version: "1.0.0",
-      activeResourcePlugins: activeResourcePlugins,
-    } as AzureSolutionSettings;
-    const result = await AADPlugin.generateArmTemplates(pluginContext);
+    return;
+    // const activeResourcePlugins = [
```

Comment:
Why commented?

Reply:
since AAD plugin has no bicep files, so this all unit test need to be deleted. later will delete this file.
task : https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12554521